v3/src/components/data-display/hooks/use-connecting-lines.ts

```diff
@@ -0,0 +1,190 @@
+import { curveLinear, line, select } from "d3"
+import { tip as d3tip } from "d3-v6-tip"
+import { useCallback } from "react"
+import { useDataConfigurationContext } from "./use-data-configuration-context"
+import { PixiBackgroundPassThroughEvent, PixiPoints } from "../../graph/utilities/pixi-points"
+import { t } from "../../../utilities/translation/translate"
+import { isGraphDataConfigurationModel } from "../../graph/models/graph-data-configuration-model"
+import { IConnectingLineDescription, transitionDuration } from "../data-display-types"
+import { useMapModelContext } from "../../map/hooks/use-map-model-context"
```

Comment:
Ideally, these map/graph dependencies would be refactored away. I will add a commit to demonstrate.